google/cloud/artifactregistry/README.md

```diff
@@ -3,7 +3,8 @@
 :construction:
 
 This directory contains an idiomatic C++ client library for the
-[Artifact Registry API][cloud-service-docs], a service to Store and manage build artifacts in a scalable and integrated service built on Google infrastructure.
+[Artifact Registry API][cloud-service-docs]; store and manage build
```

Comment:
Semicolons are pretentious. I would prefer any of:

* s/; store/. Use this (service|API) to store/
* s/; store/. Store/
* s/; store/, a service to store/

Reply:
Fixed.